tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -206,6 +235,7 @@ private:
    std::string fContainerLabel{};
    std::shared_ptr<RDaosPool> fPool;
    ObjClassId_t fDefaultObjectClass{OC_SX};
+   uint32_t fDefaultCageSize = 16 * 64 * 1024;
```

Comment:
This shouldn't be here, no?, i.e. "caging" is a concept defined in the high-level layer "RPageStorageDaos`.  Otherwise, I think it's unused here.
The bookkeeping of the current "cage" size should be kept in `RPageStorageDaos` instead.  If you agree, don't forget to also remove the getter/setter below.

Reply:
I agree, this is misplaced - it can be a private member of `RPageSinkDaos`. If `RNTupleLocator` retains the cage size for each page, `RPageSourceDaos` will not need this maximum size (nor will the anchor).